### PR TITLE
[DC-3175] Update the vocabulary QC notebook

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py [hunk 1]
```diff
@@ -85,7 +93,8 @@
 UNION ALL
 
 SELECT
-c.concept_code
+c.concept_code,
+'recently_deprecated' as status
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/vocabulary_upload_qc.py:97]: The status field is really helpful!

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py [hunk 2]
```diff
@@ -98,27 +107,38 @@
...
+WHERE vocabulary_id in ('AoU_Custom', 'AoU_General')
+OR concept_code IN ('AOU generated') 
+''')
+query = tpl.render(vocabulary_dataset_old=vocabulary_dataset_old,
+                   vocabulary_dataset_new=vocabulary_dataset_new,
+                  custom_concepts=custom_concepts)
+df = execute(client, query, max_rows=True)
+
+display(df)
+print('This check passes if the total is ' + str(len(custom_concepts)))
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/vocabulary_upload_qc.py:132]: This statement may be hard to miss. Maybe an IF ELSE with COUNT(*)>{str(len(custom_concepts))}?

**Review by nishanthpp93 — COMMENTED** [review]
Hey Brenda, sorry I thought I had submitted the review for this but it was pending. Overall the changes look good and works but added some suggestions. Let me know if you have any questions.

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py [hunk 3]
```diff
@@ -32,8 +27,19 @@
...
+old_vocabulary = ''
+new_vocabulary = ''
+run_as = ''
 # -
 
+# These are the AoU_Custom and AoU_General concepts. Look for added concepts in the aou_vocab/CONCEPT.csv. Link Above. 
+custom_concepts = [2000000000,2000000001,2000000002,2000000003,2000000004,2000000005,
+                   2000000006,2000000007,2000000008,2000000009,2000000010,2000000011,
+                   2000000012,2000000013,2100000000,2100000001,2100000002,2100000003,
+                   2100000004,2100000005,2100000006,2100000007]
```
[the remark this answers is not in the document]
Comment (brendagutman) [reply data_steward/analytics/cdr_ops/vocabulary_upload_qc.py]: Yes, I see what you mean. Thanks for the suggestion.

File: data_steward/analytics/cdr_ops/vocabulary_upload_qc.py (hunk 2 above)
Comment (brendagutman) [reply data_steward/analytics/cdr_ops/vocabulary_upload_qc.py]: I changed a few things to make this easier to see the pass/fail status and troubleshooting. Let me know if you have any more suggestions on these changes. Thanks!

**Comment by brendagutman** [comment]
Replacing this PR and branch with [#1585 ](https://github.com/all-of-us/curation/pull/1585)

**Comment by brendagutman** [comment]
Closing this PR. Another [replaces it](https://github.com/all-of-us/curation/pull/1585).

**Pull request closed without merging**